xml/docu_styleguide.structure.xml

```diff
@@ -588,17 +588,36 @@ data  home  lost+found  opt     run   srv      tmp&lt;/screen&gt;</screen>
       into lines can also be helpful for aligning callouts with the right option.
      </para>
     </listitem>
-    <!-- Comment this for the moment... - sknorr, 2017-03-06 -->
-    <!-- <listitem>
+    <listitem>
+     <para>
+      Under certain conditions, you can combine multiple commands within a
+      <tag class="emptytag">screen</tag> element:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        All commands must be explained in an unambiguous way before the <tag
+        class="emptytag">screen</tag> element.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        All commands combined within the <tag class="emptytag">screen</tag>
+        element must be strictly consecutive and non-optional.
+       </para>
+      </listitem>
+     </itemizedlist>
+     <para>
+      In all other cases, do not show multiple commands within the same
+      <tag class="emptytag">screen</tag> element and use a procedure instead.
```

Comment:
Do we really want to make procedure a requirement? I think the alternative for a screen is multiple screens, not necessarily a procedure. A procedure is for seqsuential tasks, but we explicitly mention the case where commands are not strictly consecutive.

Reply:
```suggestion
      In all other cases, do not combine multiple commands within one
      <tag class="emptytag">screen</tag> element. Instead, use multiple
      <tag class="emptytag">screen</tag> elements instead. For example,
      as part of a multi-step procedure.
```